release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -2708,3 +2708,59 @@ Issued: 2020-09-30
+and bug fixes included in the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2020:3843[RHBA-2020:3843] advisory.
+The RPM packages included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2020:3844[RHBA-2020:3844] advisory.
+
+Space precluded documenting all of the container images for this release in the
+advisory. See the following article for notes on the container images in this
+release:
+
+link:https://access.redhat.com/solutions/5479921[{product-title} 4.5.14 container image list]
+
+[id="ocp-4-5-14-bug-fixes"]
+==== Bug fixes
+
+* With this release, the value of each logging level is documented in the logging field of the `imageregistry` API.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1843244[BZ#1843244])
+
+* Previously, a mismatch between the version and the database sometimes caused problems when restoring a pod from the most recent image. With this release, the configuration YAML file for the pod is copied with the backup to avoid causing a mismatch.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1877930[BZ#1877930])
+
+* Previously, if a pod referred to an invalid image reference, the pruner job caused the Image Registry Operator to enter a degraded state, which blocked upgrades. To be able to upgrade, users had to remove the pods that caused the issue and wait for the next pruning occurrence, or suspend the pruner job. With this release, a metric and alert have been added to indicate when the pruner job fails, and the pruner status no longer impacts the Operator status.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1879176[BZ#1879176])
+
+* Previously, Kubernetes dependencies for the Cluster DNS Operator in the {product-title} 4.5 branch were out of date. With this release, the Cluster DNS Operator dependencies are updated from Kubernetes 0.18.0-rc2 to v0.18.9.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1880311[BZ#1880311])
+
+* Previously, Kubernetes dependencies for the Cluster Ingress Operator in the {product-title} 4.5 branch were out of date. With this release, the Cluster Ingress Operator dependencies are updated from Kubernetes 0.18.3 to v0.18.9.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1880315[BZ#1880315])
+
+* Previously, a watch cache in the Kuberenetes API was initialized from the global revision (`etcd`) and could remain for an undefined period if no changes were made. This behavior sometimes led to a situation in which a client got a resource version (RV) from a server that had observed a newer RV, disconnected from it due to a network error, and reconnected to a server that is behind, resulting in "Too large resource version" errors. With this release, the reflector is fixed so that it can recover from these errors, and Operators that use the `client-go` library for getting notifications from the server can recover and make progress upon receiving the errors.
```

Comment:
s/that is behind/that was behind

Seems past tense might make more sense here, since we're talking about what happened previously. Disregard if I'm misunderstanding :slightly_smiling_face: